api/src/main/java/org/openmrs/Order.java

```diff
@@ -363,6 +363,38 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated before or on the current date
+	 * @since 2.0
+	 * @see #isActivated(java.util.Date)
```

Comment:
I wouldn't write this test myself, but I can see the argument for doing so. @wluyima can decide whether this is required or not.